File: problema05/sol_java/Ejercicio05/src/ejercicio05/Ejercicio05.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package ejercicio05;

import java.util.Scanner;

/**
 *
 * @author DETPC
 */
public class Ejercicio05 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        int x;
        int y;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingrese el valor de X: ");
        x = scanner.nextInt();
        System.out.println("Ingrese el valor de Y: ");
        y = scanner.nextInt();

        if (x > 0 && y > 0) {
            System.out.println("El punto (" + x + ", " + y + ") se encuentra en el primer cuadrante.");
        } else {
            if (x < 0 && y > 0) {
                System.out.println("El punto (" + x + ", " + y + ") se encuentra en el segundo cuadrante.");
            } else {
                if (x < 0 && y < 0) {
                    System.out.println("El punto (" + x + ", " + y + ") se encuentra en el tercer cuadrante.");
                } else {
                    if (x > 0 && y < 0) {
                        System.out.println("El punto (" + x + ", " + y + ") se encuentra en el cuarto cuadrante.");
                    } else {
                        if (x == 0 && y == 0) {
                            System.out.println("El punto (" + x + ", " + y + ") se encuentra en el origen.");
                        } else {
                            if (x == 0 && y != 0) {
                                System.out.println("El punto (" + x + ", " + y + ") se encuentra sobre el eje Y.");
                            } else {
                                if (x != 0 && y == 0) {
                                    System.out.println("El punto (" + x + ", " + y + ") se encuentra sobre el eje X.");
                                }
                            }
                        }
                    }
                }
            }
        }
    }
}
    

File: problema05/sol_python/Ejercicio05.py
x = int(input("Ingrese el valor de X: "))
y = int(input("Ingrese el valor de Y: "))
if x > 0 and y > 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra en el primer cuadrante.")
elif x < 0 and y > 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra en el segundo cuadrante.")
elif x < 0 and y < 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra en el tercer cuadrante.")
elif x > 0 and y < 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra en el cuarto cuadrante.")
elif x == 0 and y == 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra en el origen.")
elif x == 0 and y != 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra sobre el eje Y.")
elif x != 0 and y == 0:
    print("El punto (" + str(x) + ", " + str(y) + ") se encuentra sobre el eje X.")